make-images.ipynb

```diff
@@ -128,12 +130,124 @@
     "!python sendfile_osm_oauth_protector/oauth_cookie_client.py -u \"{osm_user.value}\" -p \"{osm_password.value}\" -c \"https://osm-internal.download.geofabrik.de/get_cookie\" -f netscape -o geofabrik_cookie.txt;wget --max-redirect 0 --load-cookies geofabrik_cookie.txt {file_url.value}"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "78358b23-cf0b-4890-83bc-70ead669b77d",
+   "metadata": {},
+   "source": [
+    "Once this is finished, you can move down to **Starting to make comparison images**!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "66cc684e-90e9-4ccb-a851-9315aaac5362",
+   "metadata": {},
+   "source": [
+    "#### 2.1 Option b): Using an existing Geofabrik cookie from your browser. \n",
+    "\n",
+    "This method allows you to paste the cookie that Geofabrik uses to check if you're allowed to download a file. As a result, **this method does not need to enter your OSM username/password**. The downside is that it's slightly more complicated, as it requires finding the cookie value in your web browser. \n",
+    "\n",
+    "To use this option, do the following: \n",
+    "\n",
+    "1. Head to the [OSM internal Geofabrik download page](https://osm-internal.download.geofabrik.de/)\n",
+    "2. Log-in on that page with your existing OSM credentials, this will create the necessary cookie. \n",
+    "3. Find the cookie in your web-browser: If you're using Firefox, press `F12` on your keyboard to open the web developer tools\n",
+    "4. Go to the tab called `Storage` in the bar that opens at the bottom of your browser\n",
+    "5. On the bottom-left, you will see `Cookies`, click on it and select the cookie that starts with `https://osm-internal.download.geofabrik.de`.\n",
```

Comment:
```suggestion
    "2. Log in on that page with your existing OSM credentials, this will create the necessary cookie. \n",
    "3. Find the cookie in your web browser: If you're using Firefox, press `F12` on your keyboard to open the web developer tools\n",
    "4. Go to the tab called `Storage` in the bar that opens at the bottom of your browser\n",
    "5. On the bottom left, you will see `Cookies`, click on it and select the cookie that starts with `https://osm-internal.download.geofabrik.de`.\n",
```